Platform/AlderlakeBoardPkg/CfgData/CfgData_PepConstraints.yaml

```diff
@@ -277,5 +285,5 @@
         length       : 0x01
         value        : 0x0
     - Dummy        :
-        length       : 0x0
+        length       : 0x4
```

Comment:
0x3?

Reply:
Actually, 0x1 should be sufficient, will change it.